sites-config/ci_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "artifactStructure": {
+      "assets": [
+          {
+              "root": "test-site/public"
+          }
+      ],
+      "hbsTemplates": []
+  },
+  "base_image_tag": "node-12.18",
+  "dependencies": {
+      "installDepsCmd": "npm install",
+      "requiredFiles": []
+  },
+  "buildArtifacts": {
+      "build_setup_cmd": "npm install",
+      "build_cmd": "./test-site/scripts/setup_for_yext_sites.sh && npm run setup-test-site && npm run build-test-site"
+  },
+  "livePreview": {
+      "serve_setup_cmd": "echo serve setup cmd",
+      "serve_cmd": "echo serve cmd"
```

Comment:
does live preview not work if this is something like this?
```
"serve_setup_cmd": "cd test-site/ && npx jambo build",
"serve_cmd": npx serve -l 8080 test-site/public/ &
```

Reply:
I can give it a shot

Reply:
This doesn't work because of the directory rename and some other issues. I'll defer to another item per Tom's suggestion